i18n.json

```
@@ -93,15 +93,15 @@
   { "term": "CLASS", "wAR": "مصنف", "wEN": "class" },
   { "term": "CLINET", "wAR": "عميل", "wEN": "client" },
   { "term": "CLONE", "wAR": "استنساخ", "wEN": "clone" },
-  { "term": "CODE", "wAR": "شيفرة", "wEN": "code" },
+  { "term": "CODE", "wAR": "شفرة", "wEN": "code" },
```

!شفرة : [قد يحدث سوء فهم وتأويلها شفرة الحلاقة](https://www.google.com/search?q=%D8%B4%D9%81%D8%B1%D8%A9) 
```suggestion
  { "term": "CODE", "wAR": "شيفرة", "wEN": "code" },
```